src/loader/load.cc

```diff
@@ -49,9 +52,13 @@ timetable load(std::vector<std::pair<std::string, loader_config>> const& paths,
       if (!is_in_memory) {
         log(log_lvl::info, "loader.load", "loading {}", path);
       }
+      auto const progress_tracker = utl::get_active_progress_tracker();
+      progress_tracker->context(std::string{tag});
       try {
         (*it)->load(local_config, src, *dir, tt, bitfields, a, shapes);
+        progress_tracker->context("");
```

Comment:
we would also have utl RAII stuff for this (to make sure that once the scope exits this code is executed, no matter what)